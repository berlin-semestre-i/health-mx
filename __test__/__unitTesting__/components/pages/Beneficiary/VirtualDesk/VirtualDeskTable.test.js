import React from 'react'
import { shallow, mount } from 'enzyme'
// eslint-disable-next-line max-len
import DefaultTable from '../../../../../../components/pages/Beneficiary/VirtualDesk/Table/index'

describe('Default table', () => {
  it('renders', () => {
    const component = shallow(<DefaultTable />)
    expect(component).toMatchSnapshot()
  })
})