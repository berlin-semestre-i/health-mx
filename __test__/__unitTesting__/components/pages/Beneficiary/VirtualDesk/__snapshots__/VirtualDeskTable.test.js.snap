// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default table renders 1`] = `
<Table__DeskTable
  striped={true}
>
  <TableHeader
    as="thead"
  >
    <TableRow
      as="tr"
      cellAs="td"
    >
      <Table__TableHeaderCell>
        Parentesco
      </Table__TableHeaderCell>
      <Table__TableHeaderCell>
        Nombre
      </Table__TableHeaderCell>
      <Table__TableHeaderCell>
        CURP
      </Table__TableHeaderCell>
      <Table__TableHeaderCell>
        Sexo
      </Table__TableHeaderCell>
      <Table__TableHeaderCell>
        Edad
      </Table__TableHeaderCell>
      <Table__TableHeaderCell>
        Situaci√≥n
      </Table__TableHeaderCell>
    </TableRow>
  </TableHeader>
  <TableBody
    as="tbody"
  />
</Table__DeskTable>
`;
